src/SFA.DAS.AssessorService.Web/Views/Application/StandardApplications.cshtml

```diff
@@ -11,6 +11,9 @@
             <h1 class="govuk-heading-xl">
                 Select application
             </h1>
+            <form asp-action="StartApplication" asp-controller="Application" method="post">
+                <button class="govuk-button" type="submit">Start Application</button>
```

Comment:
I think on the prototype this has changed to "Start New Application"

Reply:
Its does - Fixed
NB - we need a way of highlighting prototype changes